atlas/commands/bootstrap/templates/Makefile.gotmpl

```diff
@@ -12,14 +12,7 @@ IMAGE_VERSION		?= $(USERNAME)-dev-$(GIT_COMMIT)
 SERVER_BINARY 		:= $(BUILD_PATH)/server
 SERVER_PATH		:= $(PROJECT_ROOT)/cmd/server
 SERVER_IMAGE		:= {{ if .Registry }}$(IMAGE_REGISTRY)/{{ end }}{{ .Name }}-server
-SERVER_DOCKERFILE	:= $(DOCKERFILE_PATH)/Dockerfile.server
-{{ if .WithGateway }}
-# configuration for gateway binary and image
-GATEWAY_BINARY		:= $(BUILD_PATH)/gateway
-GATEWAY_PATH		:= $(PROJECT_ROOT)/cmd/gateway
-GATEWAY_IMAGE		:= {{ if .Registry }}$(IMAGE_REGISTRY)/{{ end }}{{ .Name }}-gateway
-GATEWAY_DOCKERFILE 	:= $(DOCKERFILE_PATH)/Dockerfile.gateway
-{{ end }}
+SERVER_DOCKERFILE	:= $(DOCKERFILE_PATH)/Dockerfile
```

Comment:
Nitpick: the formatting is a little off here. It just needs a newline.

[Here's](https://github.com/ZachEddy/atlas-cli/blob/example/example/Makefile#L16) how it currently gets rendered.